### DISCUSSION TITLE
SwiftTerm submodule fix

### DISCUSSION THREAD

**Discussion by migueldeicaza** [migueldeicaza/SwiftTerm#159] [General]
Hello,

Trying forums now.

This is not really a bug report, but a notification.

I noticed that almost every fork was dealing with the SwiftSH.binaries submodule which uses my git URL, and folks were struggling with that one, leading to an unpleasant experience.

I have updated SwiftTerm to not use Git submodules, and instead use XCFramework, and that should address your problems. You might want to merge these changes into your forks.

@mickuehl
@Baw-Appie
@steev
@JacopoMangiavacchi
@yonihemi
@playstonex
@dschaefer
@vaguilar
@rpolenthon

**Comment by steev** [comment]
Thank you! I've updated my fork as well 👍
